seqr/views/utils/terra_api_utils.py

```diff
@@ -172,15 +192,11 @@ def user_get_workspace_access_level(user, workspace_namespace, workspace_name, m
         logger.info('Terra API cache hit for: GET {} {}'.format(path, user))
         return r
 
-    try:
-        r = _user_anvil_call('get', path, user)
-    # TerraNotFoundException is handled to return empty perms to allow users to work with local projects when Terra is not available
-    # It should be taken out when local access is deprecated.
-    except TerraNotFoundException as et:
-        logger.warning(str(et))
-        return {}
+    # Exceptions are handled to return an empty result for users who have no permission to access the workspace
+    r = _user_anvil_call('get', path, user, handle_errors=True)
```

Comment:
Handling the Terra exceptions silently to the user is not so user-friendly. It is better to show a warning to the users. Maybe we can add this feature later.

Reply:
this is maintaining current behavior. We expect many users not to have access to these things, and therefore the expected response is an error